integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/utils/FileDownloaderTest.java

```diff
@@ -49,19 +51,26 @@ public void tearDown(Method method) {
         new FileDownloader().cleanup();
     }
 
+    public DefaultConnectionConfigurator createConnectionConfigurator() {
```

Comment:
I improved the `FileDownloader` API to set an `ConnectionConfigurator` which is basically just a `Consumer<URLConnection>` which gives more control for each download. The reason is described below.